base.ts

```
@@ -0,0 +1,307 @@
+    url.searchParams.set("parentUrl", window.location.origin);
+
+    this.url = url.toString();
+
+    try {
+      this.sendMessage.navigate({
+        pathname: `${url.pathname}${url.search}`,
+      });
+    } catch (e) {
+      console.error("Error redirecting LucraSports", e);
+    } finally {
+      return this;
```

Biome (2.4.9)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)